python/README.md

```diff
@@ -0,0 +1,32 @@
+# Apache Spark
+
+Spark is a fast and general cluster computing system for Big Data. It provides
```

Comment:
I see. And I'm guessing we can't/don't want to somehow reference the README in the root directory? (Perhaps even with a symlink, if necessary...)